resolwe/process/fields.py

```diff
@@ -582,6 +635,18 @@ def entity_name(self):
         """Entity name."""
         return self._get("__entity_name")
 
+    @property
+    def relations(self):
+        """Get this data's relations."""
+        relations = set()
+        for relation in [
+            RelationDescriptor.from_dict(data) for data in self._field.process.relations
+        ]:
+            for partition in relation.partitions:
+                if partition.entity_id == self.entity_id:
+                    relations.add(relation)
```

Comment:
It would make sense to initialize `RelationDescriptor` here, not in the `for` loop.

Reply:
True, will fix.